boxsdk/object/folder.py

```diff
@@ -448,9 +448,12 @@ def create_web_link(self, target_url, name=None, description=None):
             web_link_attributes['name'] = name
         if description is not None:
             web_link_attributes['description'] = description
-        box_response = self._session.post(url, data=json.dumps(web_link_attributes))
-        response = box_response.json()
-        return self.translator.translate('web_link')(session=self._session, object_id=response['id'])
+        response = self._session.post(url, data=json.dumps(web_link_attributes)).json()
+        return self.translator.translate('web_link')(
```

Comment:
We should use the `response['type']` value when translating, rather then hardcoding it